store/todos.js

```diff
@@ -1,5 +1,12 @@
 import { uuid } from 'uuidv4';
```

Comment:
nitpicking: this warning was shown in terminal, which might be related to this import.

```bash
(node:10626) DeprecationWarning: uuidv4() is deprecated. Use v4() from the uuid module instead.   09:04:24
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Reply:
Okay. Thank you